runtime/registry.go

```diff
@@ -55,6 +56,19 @@ func (r *Runtime) CreateInstance(ctx context.Context, inst *drivers.Instance) er
 		return fmt.Errorf("failed to prepare instance: %w", err)
 	}
 
+	file, err := repoStore.Get(ctx, inst.ID, "rill.yaml")
+	// ignoring fs.PathError since rill.yaml may not be present for older projects
+	var pathError *fs.PathError
+	if err != nil && !errors.As(err, &pathError) {
+		return err
+	}
+
+	env, err := drivers.NewEnvVariables(ctx, file, envString)
+	if err != nil {
+		return fmt.Errorf("failed to parse env variables %w", err)
+	}
+	inst.Env = &env
```

Comment:
This is our first foray into parsing `rill.yaml`. Let's setup code for reading and parsing it in `compilers/rillv1beta` (another very partially implemented package), instead of in `drivers`.

Reply:
Could look something like:

```go
c := rillv1beta.New(repoStore, inst.ID)
proj := c.ProjectConfig()
inst.DefaultEnv = proj.DefaultEnv
```